mkdocs.yml

```diff
@@ -69,10 +63,16 @@ extra_javascript:
     - https://polyfill.io/v3/polyfill.min.js?features=es6
     - https://unpkg.com/mermaid/dist/mermaid.min.js
     - https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js
+    - javascripts/refresh_on_toggle_dark_light.js
 
 plugins:
     - search
-    - mermaid2
+    - mermaid2:
```

Comment:
Ah wait, no. You're not using [our Mermaid.js integration](https://squidfunk.github.io/mkdocs-material/reference/diagrams/#configuration) but [a third-party plugin](https://github.com/fralau/mkdocs-mermaid2-plugin).

Reply:
Thanks for the quick feedback.
This is interesting! 
Could we get rid of the 3rd party plugin `mkdocs-mermaid2-plugin` with mkdocs-material version 7.3.6 or do we need to update to the latest version beforehand?

```
pip list 
#...
mkdocs                     1.3.0
mkdocs-material            7.3.6
mkdocs-material-extensions 1.1.1
mkdocs-mermaid2-plugin     0.6.0
```

Reply:
Mermaid.js integration is available as of 8.2.0, so you would at least need to update to this version. Please note that we're not providing support for older versions, so its advisable to try to stay on the latest major version.

However, given that you have access to Insiders according to https://github.com/LoopKit/loopdocs/pull/573#issuecomment-1426363909 you could switch to the last version of Insiders that was built on 7.3.6, which also includes the integration. It was originally added in [insiders-1.15.0](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#1.15.0), which is based on [6.2.3](https://squidfunk.github.io/mkdocs-material/changelog/#6.2.3). Thus, [7.3.6+insiders-3.2.3](https://github.com/squidfunk/mkdocs-material-insiders/releases/tag/7.3.6-insiders-3.2.3) might be a temporary option.

Reply:
👍🏻 
I have upgraded to the latest version on a [branch](https://github.com/ebouchut/loopdocs/tree/upgrade_mkdocs_material) to make sure I set things up correctly.
We therefore could benefit from the most recent features and fixes 😄.